src/main/java/org/neo4j/shell/tools/imp/format/kryo/KryoSerializationTypes.java

```diff
@@ -0,0 +1,12 @@
+package org.neo4j.shell.tools.imp.format.kryo;
+
+/**
+ * Created by efulton on 5/5/16.
+ */
+public enum KryoSerializationTypes {
```

Comment:
could we make these enums strings? so that someone else with another tool could import the data too even if they don't have shell tools